tools/schemacode/bidsschematools/render/tables.py

```diff
@@ -500,55 +499,48 @@ def make_metadata_table(schema, field_info, src_path=None, tablefmt="github"):
     return table_str
 
 
-def make_subobject_table(schema, object_tuple, field_info, src_path=None, tablefmt="github"):
-    """Create a table of properties within an object.
+def make_subobject_table(
+    schema: Namespace,
+    object_name: str,
+    src_path: ty.Optional[str] = None,
+    tablefmt: str = "github",
+):
+    """Create a metadata table (markdown) based on the properties of an object
 
     Parameters
     ----------
-    schema : Namespace
+    schema : dict
```

Comment:
```suggestion
    schema : Namespace
```

Think this got jumbled, did you mean Namespace here instead of dict per line 503?